### PR TITLE
[Habpanel] Selection widget 5 columns not possible #105

### PR DESCRIPTION
**Pull request by Alluuu**
Proposed fix for the selection widget 5 columns possibility referenced in issue #105

### CODE REVIEW

**Comment by ghys** [comment]
Thanks! I suppose you didn't test it? - the SASS files have to be compiled with `gulp sass` to be integrated in the final minified CSS file. This is not handled by a build step currently for HABPanel therefore the new compiled CSS has to be included with new PRs when the styles are altered.

Also could you please review the [contribution guidelines](https://www.openhab.org/docs/developer/contributing.html#sign-your-work) regarding signing your work.

File: bundles/org.openhab.ui.habpanel/web/app/widgets/selection/selection.widget.js [hunk 1]
```diff
@@ -121,7 +121,7 @@
         vm.square = vm.columns > 1 && !widget.no_squares;
 
         vm.gridClassMode = (vm.mobile_mode) ? 'sm' : 'xs';
-        vm.gridClass = 'col-' + vm.gridClassMode + '-' + (12 / vm.columns).toString();
+        vm.gridClass = 'col-' + vm.gridClassMode + '-' + (vm.colums == 5 ? '5ths' : (12 / vm.columns).toString());;
```
Comment (ghys) [inline bundles/org.openhab.ui.habpanel/web/app/widgets/selection/selection.widget.js]:
```suggestion
        vm.gridClass = 'col-' + vm.gridClassMode + '-' + (vm.columns == 5 ? '5ths' : (12 / vm.columns).toString());;
```
?

**Comment by Alluuu** [comment]
Thanks for the feedback.

I figured that the css files would be compiled during some automated build, I did this code change in github with the browser editing as I currently don't have a java dev environment at hand.

Is it possible for someone, who has everything ready to rebuild in order to generate changed compiled css?

Also how could I have this change take effect on my OH 2.4 install?

**Comment by ghys** [comment]
@Alluuu I have integrated your code in #116 - this will automatically close this PR once #116 is merged.
After that you can download the .jar from https://ci.openhab.org/job/openHAB-WebUI/lastSuccessfulBuild/org.openhab.ui.bundles$org.openhab.ui.habpanel/, uninstall your current HABPanel with Paper UI and put the new .jar in your `addons` folder (check the docs).
No guarantee it will work with 2.4.0, quite a lot has happened since then.

**Pull request closed without merging**